src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/InfrastructureFormula.kt

```diff
@@ -40,6 +43,7 @@ class InfrastructureFormula<out T : VirtualUsers> private constructor(
 ) {
     private val logger: Logger = LogManager.getLogger(this::class.java)
 
+    @Deprecated("Use InfrastructureFormula.Builder instead.")
     constructor(
```

Comment:
AwsDatasetModification is using it, consider removing the usage

Reply:
- [ ] Convert `AwsDatasetModification`'s usage of `InfrastructureFormula` to it's `Builder`